en/orm/database-basics.rst

```diff
@@ -372,6 +369,25 @@ automatically convert input parameters from ``DateTime`` instances into a
 timestamp or formatted datestrings. Likewise, 'binary' columns will accept file
 handles, and generate file handles when reading data.
 
+.. _datetime-type:
+
+DateTime Type
+-------------
+
+.. php:class:: Type\DateTimeType
+
+Maps to a timezone naive ``DATETIME`` column type. In PostgreSQL, and SQL Server
```

Comment:
naive or native? :)

Reply:
I had just moved the sentence. Fixed in new PR.